src/runtime/cli/kata-exec.go

```diff
@@ -54,12 +56,12 @@ var kataExecCLICommand = cli.Command{
 	Usage: "Enter into guest by debug console",
 	Flags: []cli.Flag{
 		cli.StringFlag{
-			Name:  paramKataMonitorAddr,
-			Usage: "Kata monitor listen address.",
+			Name:  paramRuntimeNamespace,
+			Usage: "Namespace that containerd or CRI-O are using for containers. (Default: k8s.io, only works for containerd)",
```

Comment:
Maybe this should state "`Containerd namespace ...`" to avoid confusing with all the other namespace types?

Reply:
In fact, we also support CRI-O ...